src/data/index.ts

```diff
@@ -38,6 +38,7 @@ import { metaColonyResolvers } from './resolvers/metacolony';
 import { eventsResolvers } from './resolvers/events';
 import { recoveryModeResolvers } from './resolvers/recovery';
 import { extensionsResolvers } from './resolvers/extensions';
+import { motionsResolvers } from './resolvers/motions';
```

Comment:
I created the same file, just that mine is called `motion` :P

But don't worry, I'll merge them in my PR